config/locales/views/saved_annotations/nl.yml

```
@@ -2,3 +2,13 @@ nl:
   saved_annotations:
     index:
       title: Opgeslagen opmerkingen
+    show:
+      linked_submissions: Gekoppelde indieningen
```

```suggestion
      linked_submissions: Gekoppelde oplossingen
```

I think we use "oplossingen" or "ingediende oplossingen", but never "indieningen" on Dodona.